src/hooks/useConnectOwnedVenues.tsx

```diff
@@ -0,0 +1,88 @@
+import { useMemo, useCallback } from "react";
+import { ReactHook } from "types/utility";
+import { AnyVenue } from "types/venues";
+
+import { WithId, withId } from "utils/id";
+import { ownedVenuesDataSelector } from "utils/selectors";
+
+import { useFirestoreConnect, isLoaded } from "hooks/useFirestoreConnect";
+import { useSelector } from "hooks/useSelector";
+import { useUser } from "hooks/useUser";
+
+export const useConnectOwnedVenues = (userId?: string) => {
+  useFirestoreConnect(
+    userId
+      ? {
+          collection: "venues",
+          where: [["owners", "array-contains", userId]],
+          storeAs: "ownedVenues",
+        }
+      : undefined
+  );
+};
+
+export interface UseOwnedVenuesProps {
+  currentVenueId?: string;
+}
+
+export interface UseOwnedVenuesData {
+  isLoaded: boolean;
+  isLoading: boolean;
+
+  currentVenue?: WithId<AnyVenue>;
+
+  ownedVenues: WithId<AnyVenue>[];
+  ownedVenuesIds: string[];
+
+  findVenueInOwnedVenues: (
+    searchedForVenueId: string
+  ) => WithId<AnyVenue> | undefined;
+}
+
+export const useOwnedVenues: ReactHook<
+  useOwnedVenuesProps,
+  useOwnedVenuesData
+> = ({ currentVenueId }): useOwnedVenuesData => {
```

Comment:
```suggestion
export const useOwnedVenues: ReactHook<
  UseOwnedVenuesProps,
  UseOwnedVenuesData
> = ({ currentVenueId }) => {
```

Reply:
this got fixed, but was too tired to notice I didn't type `git push` 354fa97f162e3211548b0d13dfaf429aa7932460